hudi-spark-datasource/hudi-spark/src/test/java/HoodieJavaStreamingApp.java

```diff
@@ -307,7 +304,7 @@ public int addInputAndValidateIngestion(SparkSession spark, FileSystem fs, Strin
       Thread.sleep(3000);
       waitTillNCommits(fs, numExpCommits, 180, 3);
       commitInstantTime2 = HoodieDataSourceHelpers.listCommitsSince(fs, tablePath, commitInstantTime1).stream().sorted().findFirst().get();
-      LOG.info("Second commit at instant time :" + commitInstantTime2);
+      log.info("Second commit at instant time :" + commitInstantTime2);
```

Comment:
We can apply parameterized logging to some places in this class

Reply:
Done!